package.json

```diff
@@ -89,8 +67,16 @@
     "diff": "^4.0.2",
     "lodash": "^4.17.11",
     "prosemirror-changeset": "^2.1.2",
-    "prosemirror-tables": "^0.7.8",
+    "prosemirror-example-setup": "^1.1.2",
```

Comment:
Our developer dependencies didn't really reflect what should be bundled – this change corrects that.